rest/rest-client/src/main/java/org/ow2/proactive/scheduler/rest/SchedulerClient.java

```diff
@@ -308,6 +308,19 @@ public JobState getJobState(JobId jobId) throws NotConnectedException, UnknownJo
         return jobInfos;
     }
 
+    @Override
+    public List<JobInfo> getjobsInfoList(List<String> jobsId)
```

Comment:
`getjobsInfoList` should be `getJobsInfoList`. This is applicable to all other methods with this name.